src/layouts/DataImportBatch__c-NPSP Data Import Batch Layout.layout

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The default button options on the page layout are weird. Any idea if we can update the defaults to show Edit, Clone, and Process Batch instead of all those weird New (Object) buttons?
![screen shot 2017-09-07 at 10 38 06 am](https://user-images.githubusercontent.com/1018198/30176934-fe38f16a-93b8-11e7-8fd0-58b675c78e46.png)

Reply:
I've just updated the quick action section to put Edit, Clone, Process Batch, and Delete first.  I left the others alone since they are Salesforce defaults, and I really didn't know what was best.  good catch by the way!